lib/gems/pending/openstack/openstack_event_monitor.rb

```diff
@@ -1,129 +0,0 @@
-# The OpentstackEventMonitor uses a plugin pattern to instantiate the correct
-# subclass as a plugin based on the #available? class method implemented in each
-# subclass
-require 'more_core_extensions/core_ext/hash'
-require 'util/extensions/miq-module'
```

Comment:
`manageiq-gems-pending` should be listed in the gemfile of `manageiq-providers-openstack` with a comment that we're using `miq-module`